scripts/tensorflow_step_image/README.md

```diff
@@ -0,0 +1,16 @@
+# A Slim Tensorflow Step Image
+
+This image is used when submitting Argo workflows to run Tensorflow/PAI Tensorflow jobs. To build this image, you should follow the below steps:
+
+1. Build the `repl` command binary and copy it under this directory.
+2. Copy `python/sqlflow_submitter` directory to this directory.
+3. Build and copy the Java parser jar file under this directory.
+
+You can checkout `Dockerfile` to find out how to build above components. After above steps, we are expecting to have below files under current directory before we can run `docker build`:
```

Comment:
It's more easy to get the `repl` and parse jar file to use the following command instead of checkout `Dockerfile`:

```bash
> docker run --rm -it -v $PWD:/opt/output` sqlflow/sqlflow cp -r /usr/local/bin/repl /opt/sqlflow-parser/parser-1.0-SNAPSHOT-jar-with-dependencies.jar /opt/output
```